packages/cactus-core/src/main/typescript/consensus-has-transaction-finality.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  ConsensusAlgorithmFamily,
+  ConsensusAlgorithmFamiliesWithTxFinality,
+  ConsensusAlgorithmFamiliesWithOutTxFinality,
+} from "@hyperledger/cactus-core-api";
+
+export function consensusHasTransactionFinality(
+  consensusAlgorithmFamily: ConsensusAlgorithmFamily,
+): boolean {
+  const isInConsensusAlgorithmFamiliesWithTxFinality = (Object.values(
+    ConsensusAlgorithmFamiliesWithTxFinality,
+  ) as string[]).includes(consensusAlgorithmFamily.toString());
+
+  const isInConsensusAlgorithmFamiliesWithOutTxFinality = (Object.values(
+    ConsensusAlgorithmFamiliesWithOutTxFinality,
+  ) as string[]).includes(consensusAlgorithmFamily.toString());
+
+  const unrecognizedConsensusAlgorithmFamily =
+    !isInConsensusAlgorithmFamiliesWithTxFinality &&
+    !isInConsensusAlgorithmFamiliesWithOutTxFinality;
+
+  if (unrecognizedConsensusAlgorithmFamily) {
+    throw new Error(
+      `Unrecognized consensus algorithm family: ${consensusAlgorithmFamily}`,
+    );
+  }
+  return isInConsensusAlgorithmFamiliesWithTxFinality;
+}
+//export defualt consensusHasTransactionFinality;
```

Comment:
@jscode017 Recommend to remove commented lines unless you really need them for some specific reason (assuming here this was just forgotten and no longer needed at all)

Reply:
My bad, forget to do a little clean up